gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageStatistics.java

```diff
@@ -19,5 +19,29 @@
 /** Statistics which are collected in GCSIO. */
 public enum GoogleCloudStorageStatistics {
   // number of objects deleted
-  OBJECT_DELETE_OBJECTS
+  OBJECT_DELETE_OBJECTS,
```

Comment:
This seems inconsistent with rest of the values

Reply:
Delete, if not used

Reply:
This is used to update to statistics of delete objects in gcsio